packages/core/test/lib/api.test.ts

```diff
@@ -119,12 +130,12 @@ describe('API', () => {
   });
 
   test('getDsn', () => {
```

Comment:
We should either adjust the name of this test or adjust this test in general.

Reply:
yeah will fix!